lib/spack/spack/test/environment.py

```diff
@@ -66,27 +66,16 @@ def miscellaneous_paths():
         '/usr/opt/lib',
         '/lib',
         '/',
-        '/usr',
+        '/usr/',
```

Comment:
let's keep both, to expand the test case (the output should of course stay the same)